### PR TITLE
Set up `ttl` queue

### LINKED ISSUE #3: Introduce `ttl` option
The introduction of the `ttl` (time to live) option will enable users to set a specific lifespan for shortened links.

### Proposed solution
- We will add a new optional parameter, `ttl`, to the link-shortening functionality. Users can specify a time in seconds for how long the shortened link should remain active.
- Additionally, we will implement a daily GitHub workflow that cleans up expired links based on their TTL.

### Implementation plan
1. Add the `ttl` option to the link-shortener, allowing users to specify a TTL for each shortened link.
2. Create a daily GitHub workflow that identifies and deletes links that have exceeded their TTL.
3. Implement storage of TTL data separately using Deno Kv.

This enhancement will provide more control over the lifespan of shortened links and improve overall link management.

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
I created a queue with the [Deno Queue](https://docs.deno.com/kv/manual/queue_overview) API with a channel of "ttl".

This allows users to define a time-to-live for their shortlink which will be deleted from the database after the desired duration has elapsed.

Resolves #3.

### CODE REVIEW

File: main.ts [hunk 1]
```diff
@@ -41,88 +54,114 @@ async function onListen() {
...
+        shorter(options)
+          .then(async (result) => {
+            // Send the success message.
+            await api.editOriginalInteractionResponse({
               botID: env.DISCORD_CLIENT_ID,
               botToken: env.DISCORD_TOKEN,
               interactionToken: interaction.token,
-              content: `Error: ${error.message}`,
+              content:
+                `Created commit [${result.message}](https://acmcsuf.com/code/commit/${result.sha})!`,
```
Comment (EthanThatOneKid) [inline main.ts]: Add some additional text if a TTL is present. Suggested by @karnikaavelumani.

Comment (EthanThatOneKid) [reply main.ts]: Resolved in <https://github.com/acmcsufoss/shorter/pull/5/commits/ae225af6f39fad4cb991f9f47932b24adb89222b>.

**Comment by EthanThatOneKid** [comment]
Progress: 
image

**Review by k4rni — APPROVED** [review]
Looks really good! Thank you for your hard work. 🐻

**Pull request merged**